plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/ConstraintDialog.java

```diff
@@ -539,7 +539,7 @@ private void cancelButtonPressEvent() {
 	/**
 	 * closes the shell and adds new constraint to the feature model if possible
 	 *
-	 * @param featureModel
+	 * @param currentModel
 	 * @param constraint
 	 */
```

Comment:
The method does not have any parameters! Changing the name of a non-existent parameter does not seem useful.